modules/oxide/src/less/theme/components/toolbar/toolbar.less

```diff
@@ -155,3 +155,23 @@
   display: flex;
   align-items: center;
 }
+
+.tox-focusable-wrapper {
+  pointer-events: none;
+
+  .tox-textfield-size {
+    pointer-events: all;
+  }
+
+  &:focus {
+    background-color: @toolbar-button-bespoke-focus-background-color;
+    border-radius: @control-border-radius;
+    border-style: solid;
+    border-width: 1px;
```

Comment:
Adding a focus only on focus makes the form item bob around when you tab between them. So I guess we should use the box shadow hack we use elsewhere to render a rectangle on top of the element.

Reply:
this is resolved now